homeassistant/components/wyoming/strings.json

```diff
@@ -0,0 +1,13 @@
+{
+  "options": {
+    "step": {
+      "init": {
+        "data": {
+          "stt_address": "STT address",
+          "tts_address": "TTS address",
+          "handle_address": "Handle address"
```

Comment:
Replace these with `host` and `port`. Use key references to load translations from the common file.